kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/NonJVMClientsST.java

```diff
@@ -27,13 +28,15 @@
 import io.kroxylicious.systemtests.templates.strimzi.KafkaNodePoolTemplates;
 import io.kroxylicious.systemtests.templates.strimzi.KafkaTemplates;
 
+import static io.kroxylicious.systemtests.TestTags.KAFKA_CLIENT_INDEPENDENT;
 import static io.kroxylicious.systemtests.k8s.KubeClusterResource.kubeClient;
 import static org.assertj.core.api.Assertions.assertThat;
 
 /**
  * The non-JVM clients system tests.
  */
 @ExtendWith(KroxyliciousExtension.class)
+@Tag(KAFKA_CLIENT_INDEPENDENT)
```

Comment:
The approach makes sense to me but I'm not sure about the naming! As the test is not independent of the client used. I think your aiming for a tag that identifies tests which don't use the Kafka Client from the class path. So maybe it should be `EXTERNAL_CLIENT` or `EXTERNAL_KAFKA_CLIENTS`.

Reply:
Yeah, I'm so bad on naming! I wanted to reflect that these tests ignore the `KAFKA_CLIENT` env variable.

Reply:
Haha we all suck at naming at your playing on hard doing in a second language! Keep up the good work!